src/components/BalanceWithTopupComponent.vue

```diff
@@ -20,14 +32,27 @@
         <div>
           <p class="font-bold">{{ $t('balance.Balance increase amount') }}</p>
           <div class="w-full flex-1">
-
-            <InputNumber v-model="topupAmount" v-bind="topupAmountAttrs" :inputProps="{
-              inputmode: 'numeric',
-              class: 'w-full'
-            }" @input="(test) => {
-                setFieldValue('Top up amount', test.value as number, errors['Top up amount'] != undefined);
-                setTouched(true)
-              }" :placeholder="$t('balance.Price')" inputId="amount" mode="currency" currency="EUR" locale="nl-NL" />
+            <InputNumber
+              mode="currency"
+              currency="EUR"
+              locale="nl-NL"
+              :placeholder="$t('balance.Price')"
+              :min="0.0"
+              :min-fraction-digits="0"
+              :max-fraction-digits="2"
+              v-model="topupAmount"
+              inputId="amount"
+              @input="
+                (test) => {
```

Comment:
maybe nice to actually change the "test" variable I once introduced when testing but never changed to "value" or smth

Reply:
Fixed, named to data